core/service/executor.cpp

```diff
@@ -22,8 +22,15 @@ limitations under the License.
 
 namespace executor{
 
+    std::string tolowerString(std::string s) {
```

Comment:
It is ok, but this function can be paraphrased by:
```
std::transform(cbegin(s), cend(s), begin(s), [](const auto c){ return std::tolower(c); });
```
Which one is more clear?
Note: `std::transform` requires `<algorithm>` header.